src/backend/base/langflow/inputs/inputs.py

```diff
@@ -289,6 +290,23 @@ class DropdownInput(BaseInputMixin, DropDownMixin, MetadataTraceMixin):
     options: list[str] = Field(default_factory=list)
 
 
+class MultiselectInput(BaseInputMixin, MultiselectMixin, MetadataTraceMixin):
```

Comment:
```suggestion
class MultiselectInput(BaseInputMixin, ListableMixin, DropDownMixin, MetadataTraceMixin):
    field_type: Optional[SerializableFieldTypes] = FieldTypes.TEXT
```

I think this should be enough to create this type.